applications/launcher/controller.cpp

```diff
@@ -301,13 +301,19 @@ void Controller::importDraftApps(){
                     }
                 }
                 file.close();
+<<<<<<< HEAD
                 auto name = app.property("name").toString();
                 app.setProperty("displayName", name);
                 if(!app.ok()){
                     qDebug() << "Invalid configuration" << name;
                     continue;
                 }
                 QDBusObjectPath path = apps.getApplicationPath(name);
+=======
+                auto name = appItem.property("name").toString();
+                appItem.setProperty("displayName", name);
+                QDBusObjectPath path = appsApi->getApplicationPath(name);
+>>>>>>> 0a3cc4e (Speed up erode list and fix issue with draft import)
```

Comment:
```suggestion
                auto name = app.property("name").toString();
                app.setProperty("displayName", name);
                if(!app.ok()){
                    qDebug() << "Invalid configuration" << name;
                    continue;
                }
                QDBusObjectPath path = apps.getApplicationPath(name);
```

Reply:
Damn I simply stupidly blind.. I've just read controller.cpp ok, but stupid idiot look that it's launcher controller.cpp. sure in process manager no conflits and it is the same :)